README.md

```diff
@@ -1,6 +1,8 @@
 # Parasut Ruby Sdk
 
-* [API Documentation](https://api.parasut.com/docs)
+* Last API version is v4
+* [API Documentation - v4](https://apidocs.parasut.com/)
+* [API Documentation - v1](https://api.parasut.com/docs)
```

Comment:
we should remove v1 support after this change

Reply:
I think also we remove v1